moto/ec2/responses/transit_gateway_route_tables.py

```diff
@@ -163,9 +184,47 @@ def search_transit_gateway_routes(self):
             <destinationCidrBlock>{{ route['destinationCidrBlock'] }}</destinationCidrBlock>
             <state>{{ route['state'] }}</state>
             <type>{{ route['type'] }}</type>
+            <state>active</state>
+            {% if route[type] != 'blackhole' %}
+            <transitGatewayAttachments>
+                <item>
+                    <resourceId>{{ route['transitGatewayAttachments']['resourceId'] }}</resourceId>
+                    <resourceType>{{ route['transitGatewayAttachments']['resourceType'] }}</resourceType>
+                    <transitGatewayAttachmentId>{{ route['transitGatewayAttachments']['transitGatewayAttachmentId'] }}</transitGatewayAttachmentId>
+                </item>
+            </transitGatewayAttachments>
+            {% endif %}
         </item>
         {% endfor %}
     </routeSet>
     <additionalRoutesAvailable>false</additionalRoutesAvailable>
 </SearchTransitGatewayRoutesResponse>
 """
+
+GET_TRANSIT_GATEWAY_ROUTE_TABLE_ASSOCIATIONS_RESPONSE = """<GetTransitGatewayRouteTableAssociationsResponse xmlns="http://ec2.amazonaws.com/doc/2016-11-15/">
+    <requestId>92fdc91d-c374-4217-b2b4-33f2fb0a2be7</requestId>
+    <associations>
+        {% for association in transit_gateway_route_table_associations %}
+        <item>
+            <resourceId>{{ association.route_table_association.resourceId }}</resourceId>
```

Comment:
Looks like `association.route_table_association` is defined as a `dict` in class `TransitGatewayRouteTable` - does the jinja templating here allow accessing dict keys as attributes via the dot syntax `.resourceId`? (I legitimately don't know the answer to this and haven't looked into this in detail - just want to double-check).